### PR TITLE
update crowdin.yml, CHANGELOG.md, it.json and cs.json files

### PR DESCRIPTION
**Pull request by camilavcoutinho**
#### What is the purpose of this pull request?

update Cs and IT translations

#### Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

**Review by alvim — APPROVED** [review]

**Review by paul0vinicius — DISMISSED** [review]
LGTM ✨

**Review by dvm1984 — CHANGES_REQUESTED** [review]
de onde surgiram essas mudancas para romeno> Estao erradas

**Review by dvm1984 — DISMISSED** [review]

**Review by dvm1984 — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,14 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+### Fixed
+
+- I18n Ro, It and Cs.
```
Comment (wdsrocha) [inline CHANGELOG.md:12]: There are no changes on `ro.json`

```suggestion
- I18n It and Cs.
```

**Review by dvm1984 — DISMISSED** [review]

**Review by wdsrocha — APPROVED** [review]
Just organized the commits and formatted the messages (no changes to the translations on my side).

**Review by dvm1984 — APPROVED** [review]

**Pull request merged**